src/super_gradients/examples/predict/segmentation_predict.py

```diff
@@ -0,0 +1,13 @@
+from super_gradients.common.object_names import Models
+from super_gradients.training import models
+
+
+model = models.get(model_name=Models.PP_LITE_T_SEG75, pretrained_weights="cityscapes")
+
+IMAGES = [
+    "https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg",
+]
```

Comment:
Not blocking, but it's nicer if the url would map to github (we updated notebooks to work like that).
You can find the images here https://github.com/Deci-AI/super-gradients/tree/master/documentation/source/images/examples